concepts/match-basics/about.md

```diff
@@ -138,11 +176,36 @@ fn toss(value: u8) {
 }
 
 fn main() {
-let x: u8 = 2;
-let y: u8 = 3;
-let z: u8 = 5;
-toss(x); // prints "you won!".
-toss(y); // prints "you can roll again!".
-toss(z); // prints "you lost...".
+    let x: u8 = 2;
```

Comment:
nit: instead of first creating local variables, and then passing those to `toss`, just pass the integer values directly:
```rust
toss(2); // prints "you won!"
...
```